submissions/askuznetsov/html-css-popup/index.html

```diff
@@ -0,0 +1,238 @@
+<!-- @format -->
```

Comment:
I think this comment is redundant here

Reply:
didn't even see it there. wow. thnx. updated